### PR TITLE
Jakarta Messaging 3.1 Plan Review

### PR DESCRIPTION
**Pull request by OndroMih**
Include the following in the PR:
- [x] A directory in the form wombat/x.y where x.y is the release major.minor version
- [x] An index page wombat/x.y/_index.md following [template](https://github.com/jakartaee/specification-committee/blob/master/spec_page_template.md)
- [x] Title should be on the form "Jakarta Wombat X.Y (under development)"
- [x] Includes a plan for the release. Usually, a couple of statements is enough. Otherwise, a link to a document describing the release.
- [x] **N/A** Includes a detailed description of any backward incompatibility (Mark with N/A and check if none)
- [x] **N/A** Declares optional features if there are any (Mark with N/A and check if none)
- [x] Includes minimum Java SE version
- [x] **N/A** Describe any updates or required changes including splitting out the TCK (Mark with N/A and check if not planned)
- [x] Link to updated release record

### CODE REVIEW

File: messaging/3.1/_index.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+title: "Jakarta Messaging Spec 3.1 (Under Development)"
```
Comment (ivargrimstad) [inline messaging/3.1/_index.md:2]: @OndroMih Looks good! Can you remove "Spec" from the title so it is conform with the previous versions?

Comment (OndroMih) [reply messaging/3.1/_index.md]: Done.

**Comment by OndroMih** [comment]
I've started the ballot today https://www.eclipse.org/lists/jakarta.ee-spec/msg01687.html. I apologize if it was premature, I tried to follow the spec process but I might have gotten it wrong. @dblevins. can you approve that it's OK to submit the plan for a ballot?

**Comment by ivargrimstad** [comment]
No worries, @OndroMih! 
I actually think it would be an improvement to the process if the projects initiated these ballots themselves. The mentoring concept has been very helpful so far to ensure the quality of the material submitted for the ballot. But I think we may have come to a maturity level, with so many examples to look at, that we may consider retire the mentors for the simple reviews (like plan reviews) and let the projects initiate these ballots. 

I'll approve this one on behalf of David, so the tracking is correct.

**Review by ivargrimstad — APPROVED** [review]

**Comment by kwsutter** [comment]
> I've started the ballot today https://www.eclipse.org/lists/jakarta.ee-spec/msg01687.html, @kwsutter . I apologize if it was premature, I tried to follow the spec process but I might have gotten it wrong. @dblevins. can you approve that it's OK to submit the plan for a ballot?

Thanks, @OndroMih, for taking the initiative.  As @ivargrimstad pointed out, the original intent of assigning Spec Committee mentors is to ensure that all the t's are crossed and i's dotted before submitting material for a vote.  Maybe we are nearing that level of maturity, especially with Plan Reviews.  Release Reviews?  Not so sure we're there yet...  In any case, thanks for initiating the ballot.  We'll need to figure out if we need any modifications in our [Operations Guide](https://jakarta.ee/committees/specification/operations/) for mentoring and approving PRs before ballot.

File: messaging/3.1/_index.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+* Bug fixes as they arise during the release cycle
+* Small enhancements to take advantage of some newer Java features in the lowest supported Java version (for example repeatable annotations)
+* A standalone TCK separated from the Jakarta EE TCK so that it's easier to certify standalone implementations of Jakarta Messaging
+* Documentation clean up as identified during the release cycle
+* Any updates required to meet the Java version requirements of Jakarta EE 10
+* Any requirements identified by other specifications or the Jakarta EE 10 platform projects during the release cycle.
+
+
+[Jakarta Messaging 3.1 Release Record](https://projects.eclipse.org/projects/ee4j.jca/releases/2.1.0)
```
Comment (kwsutter) [inline messaging/3.1/_index.md:24]: Incorrect copy-and-paste?  :-)  This should point at the JMS 3.1 release record, not the JCA 2.1 release record.

**Review by kwsutter — CHANGES_REQUESTED** [review]
Minor issue that needs to be cleaned up before we merge.  Thanks!

File: messaging/3.1/_index.md (hunk 2 above)
Comment (OndroMih) [reply messaging/3.1/_index.md]: Thanks for spotting this, @kwsutter. I've fixed it now.

Of course a copy and paste error, I created the release record before I submitted the PR but let the link from the JCA plan which I used as a template.

**Pull request merged**